app/vmalert/README.md

```diff
@@ -335,6 +336,21 @@ labels:
 
 For recording rules to work `-remoteWrite.url` must be specified.
 
+#### Loading rules from remote storage
+
+`vmalert` supports remote storages GCS and S3 for loading alerting and recording rules.
+See more details in description of the `-rule` command-line flag.
+Flag `-rule` can be specified multiple times and allows mixing different filesystems for rules loading.
+For example:
+```
+-rule="/path/to/file". Path to a single file with alerting rules
+-rule="dir/*.yaml" -rule="/*.yaml" -rule="gcs://vmalert-rules/tenant_%{TENANT_ID}/prod".
+```
+`-rule` value may contain %{ENV_VAR} placeholders, which are substituted by the corresponding ENV vars. 
+
+Support of remote storage for rules loading supposed to simplify rules management procedures 
```

Comment:
It would be great to mention, that wildcards is not supported at remote storages.  And provided path must contain *ONLY* vmalert rule files. Otherwise it will fail to start.

 E.g.
```
rule = bucket/vmalert/prod
#bucket content
bucket/vmalert/prod/rule-1.yaml
bucket/vmalert/prod/rule-2.yaml
# will be loaded as well and failed to start
bucket/vmalert/prod/subdir/binary
```

 If we want to support wild cards, it could be done via:


``` go
n := index.Bytes(path,"*")

prefix := path[:n]
filePattern := path

 for _, file in files{
   if !path.Match(filePattern, file) {
      continue
}
```

Reply:
Thanks! Updated the description instead